it/connectors/HIPConnectorISpec.scala

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import config.featureSwitches.FeatureSwitching
+import connectors.parsers.AppealsParser.{BadRequest, DuplicateAppeal, UnexpectedFailure}
+import models.appeals.{AppealSubmission, CrimeAppealInformation}
+import play.api.test.Helpers._
+import utils.{HIPWiremock, IntegrationSpecCommonBase}
+
+import java.time.LocalDateTime
+import scala.concurrent.ExecutionContext
+
+class HIPConnectorISpec extends IntegrationSpecCommonBase with HIPWiremock with FeatureSwitching {
+  implicit val ec: ExecutionContext = ExecutionContext.Implicits.global
+
+  class Setup {
+    val connector: HIPConnector = injector.instanceOf[HIPConnector]
+    val correlationId: String = "corId"
+  }
+
+  val penaltyNumber: String = "1234567890"
+
+  val submission:AppealSubmission = AppealSubmission(
+    taxRegime = "VAT",
+    customerReferenceNo = "123456789",
+    dateOfAppeal = LocalDateTime.of(2020, 1, 1, 0, 0, 0),
+    isLPP = false,
+    appealSubmittedBy = "client",
+    agentDetails = None,
+    appealInformation = CrimeAppealInformation(
+      reasonableExcuse = "crime",
+      honestyDeclaration = true,
+      startDateOfEvent = "2021-04-23T00:00",
+      reportedIssueToPolice = "yes",
+      statement = None,
+      lateAppeal = false,
+      lateAppealReason = None,
+      isClientResponsibleForSubmission = None,
+      isClientResponsibleForLateSubmission = None
+    )
+  )
+
+  "submitAppeal" should {
+    "Jsonify the model and send the request and return the response with Headers" in new Setup {
+      mockSuccessfulResponse()
+      val result = await(connector.submitAppeal(submission, penaltyNumber, correlationId))
+      result.isRight shouldBe true
+    }
+
+    "return duplicated error response" in new Setup {
+      mockDuplicateSubmissionResponse()
+      val result = await(connector.submitAppeal(submission, penaltyNumber, correlationId))
+      result.isLeft shouldBe true
+      result shouldBe Left(DuplicateAppeal)
```

Comment:
Personally, I'm not sure if we need `result.isLeft shouldBe true` in each of these since `result shouldBe Left(...` is already checking that it's a Left and more